### ISSUE TITLE
[libprotobuf ERROR google/protobuf/text_format.cc:298] Error parsing text-format onnx2trt_onnx.ModelProto: 1:9: Message type “onnx2trt_onnx.ModelProto” has no field named “onnx”.

### ISSUE THREAD

**Issue by dongjinxin123** [NVIDIA/TensorRT#1589]
A clear and concise description of the bug or issue.
I run the TensorRT quick start introNoteBook 1. Introduction
I run this line
!/usr/src/tensorrt/bin/trtexec --onnx=resnet50/model.onnx --saveEngine=resnet_engine_intro.trt --explicitBatch

and meet this error

&&&& RUNNING TensorRT.trtexec [TensorRT v8200] # /usr/src/tensorrt/bin/trtexec --onnx=resnet50/model.onnx --saveEngine=resnet_engine_intro.trt
[10/22/2021-20:38:56] [I] === Model Options ===
[10/22/2021-20:38:56] [I] Format: ONNX
[10/22/2021-20:38:56] [I] Model: resnet50/model.onnx
[10/22/2021-20:38:56] [I] Output:
[10/22/2021-20:38:56] [I] === Build Options ===
[10/22/2021-20:38:56] [I] Max batch: explicit batch
[10/22/2021-20:38:56] [I] Workspace: 16 MiB
[10/22/2021-20:38:56] [I] minTiming: 1
[10/22/2021-20:38:56] [I] avgTiming: 8
[10/22/2021-20:38:56] [I] Precision: FP32
[10/22/2021-20:38:56] [I] Calibration:
[10/22/2021-20:38:56] [I] Refit: Disabled
[10/22/2021-20:38:56] [I] Sparsity: Disabled
[10/22/2021-20:38:56] [I] Safe mode: Disabled
[10/22/2021-20:38:56] [I] Strict mode: Disabled
[10/22/2021-20:38:56] [I] Restricted mode: Disabled
[10/22/2021-20:38:56] [I] Save engine: resnet_engine_intro.trt
[10/22/2021-20:38:56] [I] Load engine:
[10/22/2021-20:38:56] [I] Profiling verbosity: 0
[10/22/2021-20:38:56] [I] Tactic sources: Using default tactic sources
[10/22/2021-20:38:56] [I] timingCacheMode: local
[10/22/2021-20:38:56] [I] timingCacheFile:
[10/22/2021-20:38:56] [I] Input(s)s format: fp32:CHW
[10/22/2021-20:38:56] [I] Output(s)s format: fp32:CHW
[10/22/2021-20:38:56] [I] Input build shapes: model
[10/22/2021-20:38:56] [I] Input calibration shapes: model
[10/22/2021-20:38:56] [I] === System Options ===
[10/22/2021-20:38:56] [I] Device: 0
[10/22/2021-20:38:56] [I] DLACore:
[10/22/2021-20:38:56] [I] Plugins:
[10/22/2021-20:38:56] [I] === Inference Options ===
[10/22/2021-20:38:56] [I] Batch: Explicit
[10/22/2021-20:38:56] [I] Input inference shapes: model
[10/22/2021-20:38:56] [I] Iterations: 10
[10/22/2021-20:38:56] [I] Duration: 3s (+ 200ms warm up)
[10/22/2021-20:38:56] [I] Sleep time: 0ms
[10/22/2021-20:38:56] [I] Streams: 1
[10/22/2021-20:38:56] [I] ExposeDMA: Disabled
[10/22/2021-20:38:56] [I] Data transfers: Enabled
[10/22/2021-20:38:56] [I] Spin-wait: Disabled
[10/22/2021-20:38:56] [I] Multithreading: Disabled
[10/22/2021-20:38:56] [I] CUDA Graph: Disabled
[10/22/2021-20:38:56] [I] Separate profiling: Disabled
[10/22/2021-20:38:56] [I] Time Deserialize: Disabled
[10/22/2021-20:38:56] [I] Time Refit: Disabled
[10/22/2021-20:38:56] [I] Skip inference: Disabled
[10/22/2021-20:38:56] [I] Inputs:
[10/22/2021-20:38:56] [I] === Reporting Options ===
[10/22/2021-20:38:56] [I] Verbose: Disabled
[10/22/2021-20:38:56] [I] Averages: 10 inferences
[10/22/2021-20:38:56] [I] Percentile: 99
[10/22/2021-20:38:56] [I] Dump refittable layers:Disabled
[10/22/2021-20:38:56] [I] Dump output: Disabled
[10/22/2021-20:38:56] [I] Profile: Disabled
[10/22/2021-20:38:56] [I] Export timing to JSON file:
[10/22/2021-20:38:56] [I] Export output to JSON file:
[10/22/2021-20:38:56] [I] Export profile to JSON file:
[10/22/2021-20:38:56] [I]
[10/22/2021-20:38:56] [I] === Device Information ===
[10/22/2021-20:38:56] [I] Selected Device: NVIDIA GeForce RTX 3070 Ti
[10/22/2021-20:38:56] [I] Compute Capability: 8.6
[10/22/2021-20:38:56] [I] SMs: 48
[10/22/2021-20:38:56] [I] Compute Clock Rate: 1.8 GHz
[10/22/2021-20:38:56] [I] Device Global Memory: 7948 MiB
[10/22/2021-20:38:56] [I] Shared Memory per SM: 100 KiB
[10/22/2021-20:38:56] [I] Memory Bus Width: 256 bits (ECC disabled)
[10/22/2021-20:38:56] [I] Memory Clock Rate: 9.501 GHz
[10/22/2021-20:38:56] [I]
[10/22/2021-20:38:56] [I] TensorRT version: 8200
[10/22/2021-20:38:57] [I] [TRT] [MemUsageChange] Init CUDA: CPU +443, GPU +0, now: CPU 455, GPU 1229 (MiB)
[10/22/2021-20:38:57] [I] Start parsing network model
[libprotobuf ERROR google/protobuf/text_format.cc:298] Error parsing text-format onnx2trt_onnx.ModelProto: 1:1: Invalid control characters encountered in text.
[libprotobuf ERROR google/protobuf/text_format.cc:298] Error parsing text-format onnx2trt_onnx.ModelProto: 1:9: Message type “onnx2trt_onnx.ModelProto” has no field named “onnx”.
[10/22/2021-20:38:57] [E] [TRT] ModelImporter.cpp:731: Failed to parse ONNX model from file: resnet50/model.onnx
[10/22/2021-20:38:57] [E] Failed to parse onnx file
[10/22/2021-20:38:57] [I] Finish parsing network model
[10/22/2021-20:38:57] [E] Parsing model failed
[10/22/2021-20:38:57] [E] Failed to create engine from model.
[10/22/2021-20:38:57] [E] Engine set up failed
&&&& FAILED TensorRT.trtexec [TensorRT v8200] # /usr/src/tensorrt/bin/trtexec --onnx=resnet50/model.onnx --saveEngine=resnet_engine_intro.trt

Environment
TensorRT Version: TensorRT 8.2 EA
GPU Type: 3070Ti
Nvidia Driver Version:
CUDA Version: 11.4
CUDNN Version: 8.4.2
Operating System + Version: Ubuntu 20.04
Python Version (if applicable): 3.8
TensorFlow Version (if applicable): 2.5.0
PyTorch Version (if applicable):
Baremetal or Container (if container which image + tag):

**Comment by zyxcambridge** [comment]
GPU has cuda devices: 1
----device id: 0 info----
  GPU : NVIDIA Tegra X2 
  Capbility: 6.2
  Global memory: 7858MB
  Const memory: 64KB
  SM in a block: 48KB
  warp size: 32
  threads in a block: 1024
  block dim: (1024,1024,64)
  grid dim: (2147483647,65535,65535)

Building TRT engine.
[libprotobuf ERROR google/protobuf/text_format.cc:298] Error parsing text-format onnx2trt_onnx.ModelProto: 1:9: Message type "onnx2trt_onnx.ModelProto" has no field named "version".
Failed to parse ONNX model from file../../model/pointpillar.onnx
Enable fp16!
trt_infer: Network must have at least one output
trt_infer: Network validation failed.

**Comment by ttyio** [comment]
Hello @zyxcambridge , sounds like the onnx is invalid, can the file run with onnxruntime? thanks!

**Comment by ttyio** [comment]
close since no activity for more than 3 weeks, please reopen if you still have question, thanks!
